src/Symfony/Component/Yaml/Inline.php

```diff
@@ -127,6 +127,8 @@ public static function dump($value, int $flags = 0): string
                 return self::dumpNull($flags);
             case $value instanceof \DateTimeInterface:
                 return $value->format('c');
+            case $value instanceof \UnitEnum:
```

Comment:
if we add new things in the Yaml component, I'd really appreciate to add some tests there covering this kind of feature

Reply:
Added one [here](https://github.com/symfony/symfony/pull/40857/files#diff-b6691addb63f50427940df5222cff925ace62f86f9dd7549fb417e1315cf646bR589). Not sure if there's another one to add? This one should be enough if I'm right.